lib/zelastic/indexer.rb

```diff
@@ -17,19 +17,19 @@ def initialize(config)
       @config = config
     end
 
-    def index_batch(batch, client: nil, index_name: nil)
+    def index_batch(batch, client: nil, index_name: nil, refresh: false)
```

Comment:
@shyvy88 are there many other options we could pass that are the same format? wondering whether a has as the final params e.g. optional might be more future proof? your call

Reply:
There are many options that can be passed down, I think if we want to expose any others we can refactor to have an options hash. `refresh` is already an optional param as it has a default set.